pytest_fixtures/component/katello_certs_check.py

```diff
@@ -2,7 +2,7 @@
 from pathlib import Path
 
 import pytest
-from broker import Broker
+from broker.broker import Broker
```

Comment:
Raised a question about this in the Broker PR, so I won't repeat it further in this PR, but this is likely a blocker for me.